figures/death_curves/scripts/deathcurves_plot.r

```diff
@@ -15,32 +15,34 @@ death_df <- read.csv(death_curve_path)
 head(death_df)
 
 # death percentage
-death_df$percentage_dead_cells <- death_df$Death.Cells/(death_df$Death.Cells + death_df$Live.Cells)	*100
-death_df$treatment <- paste0(death_df$Compound," ",death_df$Dose," ",death_df$unit, " ",death_df$time, " ", death_df$`unit.1`)
-head(death_df)
+# drop the NA values
+death_df <- death_df[!is.na(death_df$Dead_cells),]
+death_df$percentage_dead_cells <- death_df$Dead_cells/(death_df$Dead_cells + death_df$Live_cells)	*100
```

Comment:
```suggestion
death_df$percentage_dead_cells <- death_df$Dead_cells/(death_df$Dead_cells + death_df$Live_cells)*100
```
Not sure if the space there is necessary so consider removing it.

Reply:
removed